packages/apps/auth0-actions/src/login.ts

```diff
@@ -37,7 +37,16 @@ async function login(email: string, password: string): Promise<User> {
     throw new WrongUsernameOrPasswordError(email, invalidCredentialsMessage);
   }
 
-  return patronRecordToUser(patronRecord);
+  const user = patronRecordToUser(patronRecord);
+  return {
+    ...user,
```

Comment:
I think it's possible for accounts to exist in Sierra with unvalidated e-mail addresses, though practically we are unlikely to encounter that situation or for it to be an issue.

Reply:
Do the patron records tell us whether they're validated? If so easy enough to account for that

Reply:
I do not know. There is nothing obvious on the [patron record](https://sandbox.iii.com/iii/sierra-api/swagger/index.html#!/patrons/Get_a_patron_by_record_ID_get_18) that indicates that. @pollecuttn may know how that's distinguished.

I wouldn't let that block  merging this though.